src/systems/filecoin_vm/sysactors/_index.md

```diff
@@ -5,10 +5,13 @@ entries:
 - init_actor
 - cron_actor
 - account_actor
+- reward_actor
 ---
 
 - There are two system actors required for VM processing:
   - [CronActor](#CronActor) - runs critical functions at every epoch
   - [InitActor](#InitActor) - initializes new actors
-- There is one more VM level actor:
+- There are two more VM level actors:
   - [AccountActor](#AccountActor) - for user accounts.
+  - [RewardActor](#RewardActor) - for block reward and token vesting
```

Comment:
nit: may be worth clarifying here that RewardActor is a system/singleton (but AccountActor is not)